polygeist/lib/Dialect/Polygeist/Transforms/LoopInternalization.cpp

```diff
@@ -1200,29 +1209,30 @@ void LoopInternalization::transform(T loop,
     sycl::AccessorType accTy =
         getAccessorType(cast<sycl::SYCLAccessorSubscriptOp>(memref));
 
-    // Get pointer to the local memory portion for each memref.
-    memref::ViewOp view =
+    // Get pointer to the shared memory portion for each memref.
+    memref::ViewOp viewOp =
         createViewOp(accTy, ValueOrUnsigned::getValue(offset, builder),
                      getGlobalOp, workGroupSize, builder, loc);
 
-    promote(memref, view, loopInfo, localIDs, builder, solver);
+    promote(memref, viewOp, loopInfo, localIDs, builder, solver);
 
     // Only increment offset when the current memref is not the last one.
     if (memref != *memrefs.rbegin()) {
-      std::variant<Value, unsigned> reqdLocalMemory =
-          getReqdLocalMemory(accTy, workGroupSize, builder);
-      offset = ValueOrUnsigned::add(offset, reqdLocalMemory, builder);
+      std::variant<Value, unsigned> reqdSharedMemory =
+          getReqdSharedMemory(accTy, workGroupSize, builder);
+      offset = ValueOrUnsigned::add(offset, reqdSharedMemory, builder);
     }
   }
   LLVM_DEBUG(llvm::dbgs() << "Promoted loop: " << loop << "\n");
 
   builder.setInsertionPoint(loop);
-  createLocalBarrier(builder);
+  createWorkGroupBarrier(builder);
   builder.setInsertionPointAfter(loop);
-  createLocalBarrier(builder);
+  createWorkGroupBarrier(builder);
 }
 
-void LoopInternalization::promote(Operation *memref, memref::ViewOp localMemory,
+void LoopInternalization::promote(Operation *memref,
+                                  memref::ViewOp sharedMemory,
```

Comment:
`localMemory` is renamed to `sharedMemory` here, but renamed to `viewOp` in the declarator.

Reply:
ty - will change

Reply:
done